src/govuk/_govuk-prototype-kit.scss

```diff
@@ -0,0 +1,12 @@
+$govuk-prototype-kit-major-version: 0 !default;
+
+$govuk-assets-path: if(
+  $govuk-prototype-kit-major-version > 12,
+  $govuk-extensions-url-context + "/govuk-frontend/govuk/assets/",
+  "/govuk/assets/"
+);
+
+$govuk-global-styles: true;
+$govuk-new-link-styles: true;
```

Comment:
These should be `!default` so that users can opt out in the Prototype Kit settings file.